modules/marketplace/manifests/apps/frappe/admin_instance.pp

```diff
@@ -35,16 +39,13 @@
   file {
     "${project_dir}/deploysettings.py":
       content => template('marketplace/apps/frappe/admin/deploysettings.py');
+
     "${project_dir}/fabfile.py":
       content => template('marketplace/apps/frappe/admin/fabfile.py');
-  }->
-  dreadnot::stack {
-    $domain:
-      require       => File["${project_dir}/deploysettings.py"],
-      instance_name => $dreadnot_instance,
-      github_url    => 'https://github.com/grafos-ml/frappe',
-      git_url       => 'git://github.com/grafos-ml/frappe',
-      project_dir   => $project_dir;
```

Comment:
Should be fine since we are going to transition to jenkins soon and this doesn't get deployed often.